enterprise/server/remote_execution/containers/docker/docker.go

```diff
@@ -160,50 +173,37 @@ func (r *dockerCommandContainer) Run(ctx context.Context, command *repb.Command,
 		}()
 	}()
 
-	r.copyContainerLogs(ctx, cid, result)
-	if result.Error != nil {
+	var stdout, stderr bytes.Buffer
+	eg := &errgroup.Group{}
+	eg.Go(func() error {
+		_, errCopy := stdcopy.StdCopy(&stdout, &stderr, hijackedResp.Reader)
+		return errCopy
+	})
+	eg.Go(func() error {
+		statusCh, errCh := r.client.ContainerWait(ctx, cid, dockercontainer.WaitConditionNotRunning)
+		select {
+		case err := <-errCh:
+			result.Error = wrapDockerErr(err, "container did not exit cleanly")
```

Comment:
`return wrapDockerErr(...)` here, since we are setting `result.Error` based on the return value below. (I think there might be a race condition otherwise)